src/Pharmacy/Services/ImportPharmacies.php

```diff
@@ -50,7 +50,8 @@ public function importPharmacies($city, $state)
             'skip' => '',
             'version' => '2.1',
         ];
-        $response = oeHttp::get('https://npiregistry.cms.hhs.gov/api/', $query);
```

Comment:
hi @rhausam , Thanks for submitting this fix. This looks very reasonable. Just place a nice comment explaining reason for this (since we may be able to remove this in the future when cms.hhs.gov updates their encryption support), and then I'll bring this into codebase.

Reply:
So, I am trying to understand two things. @rhausam you are reporting that OpenSSL in Debian is throwing an error saying that the "who's" key is too small? Yours or CMS?
@bradymiller I didn't know there was a security issue holding back this commit.

Reply:
To clarify, security is not holding this back. Guessing CMS is just not using an updated server for supporting most recent default ciphers from debian, but will let @rhausam answer your questions?

Reply:
@bradymiller Thanks.  I will add the comment as soon as I get a chance to do that today.

Reply:
@juggernautsei The error message itself wasn't exactly clear on whose key it was that was too small.  But 'dh' refers to Diffie–Hellman, and I found that the most recent OpenSSL versions have deprecated the smaller dh keys (e.g. 512-bit), which apparently is what the CMS server is still using.  This change in OpenSSL apparently was made after that protocol was identified as being too weak following the Logjam attack. Doing some further reading, someone suggested the 'ECDHE-RSA-AES256-GCM-SHA384' cipher as being mutually acceptable for servers using the older protocol and the newer versions of OpenSSL.  So I figured out how to get that cipher passed as an option to Curl, and that worked to fix the issue.

Reply:
@rhausam Thanks for the clarification. Now I am going to check my OpenSSL version on all of my Ubuntu and CentOS servers. They are all using version 1.1.1 11 Sept 2018. Which version do you have? The OpenSSL.org says that the 1.1.1 is the latest version/release.

Reply:
I'm on Debian 10 and my version is "OpenSSL 1.1.1d  10 Sep 2019".  I think the issue possibly may arise in the curl package (although I'm not certain), and what it shows is:
curl 7.64.0 (x86_64-pc-linux-gnu) libcurl/7.64.0 OpenSSL/1.1.1d zlib/1.2.11 libidn2/2.3.0 libpsl/0.20.2 (+libidn2/2.0.5) libssh2/1.8.0 nghttp2/1.36.0 librtmp/2.3
Release-Date: 2019-02-06
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL